src/emoji.js

```diff
@@ -80,3 +77,15 @@ EMOJIBASE.forEach(emoji => {
         });
     }
 });
+
+/**
+ * Strips variation selectors from a string
+ * NB. Skin tone modifers are not variation selectors:
+ * this function does not touch them. (Should it?)
+ *
+ * @param {string} str string to strip
+ * @returns {string} stripped string
```

Comment:
Does that work without the character class [...]

Reply:
bleh, no, and also it doesn't work unless you use a regex literal. Not sure how I managed to mis-test it...